src/AppBundle/Resources/views/themes/app/pagelayout.html.twig

```diff
@@ -32,7 +32,7 @@
 
 {% include '@NetgenSite/parts/facebook_api.html.twig' %}
 
-<div id="page"{% if page_css_class|default is not empty %} class="{{ page_css_class }}"{% endif %}>
+<div id="page"{% if page_css_class|default is not empty %} class="{{ page_css_class }}"{% endif %}{% if current_location_path is not empty %} data-path='["{{ current_location_path|join('","')|raw }}"]'{% endif %}>
```

Comment:
Why do you need `raw`?

Reply:
To be honest, I don't know, I think I added it by a muscle memory :) This is just an array join and it should always contain only integers, so the raw should not be required.

Reply:
:+1: 

Do you need those numbers to be strings btw? Because, if you don't, you could've just used `current_location_path|json_encode`. See https://twigfiddle.com/dbqcc4

Reply:
I don't remember exactly why, but we tried to json_encode it directly, and it lead to some problems wit the Javascript matching. Yes, I think they need to be strings :) This code is already in use on the live project, so all the surplus conditions and error checks you see in the code were the result of error avoiding that was manifesting on that project